win32ss/user/ntuser/message.c

```diff
@@ -459,8 +483,30 @@ CopyMsgToKernelMem(MSG *KernelModeMsg, MSG *UserModeMsg, PMSGMEMORY MsgMemoryEnt
         /* Copy data if required */
         if (0 != (MsgMemoryEntry->Flags & MMS_FLAG_READ))
         {
+            WCHAR lParamMsg[sizeof(StrUserKernel[0])/2 + 1] = { 0 };
```

Comment:
```suggestion
            WCHAR lParamMsg[_countof(StrUserKernel[0]) + 1] = { 0 };
```
or using `RTL_NUMBER_OF` or `ARRAYSIZE` .

Reply:
This is not big enough for the longer strings